packages/loopring_v3/test/testExchangeUtil.ts

```diff
@@ -2252,14 +2223,14 @@ export class ExchangeTestUtil {
       // Transform DA
       const transformedDa = new Bitstream();
       const size = Constants.TX_DATA_AVAILABILITY_SIZE;
-      const sizeA = 25;
-      const sizeB = 43;
-      assert.equal(sizeA + sizeB, size, "invalid transform sizes");
+      const size1 = 29;
```

Comment:
Per my understandings, TX_DATA_AVAILABILITY_SIZE(68) is the max bytes of DA in all types, what's the reason of this 29+39(before 25+43)? I think it is a separation best fits for 1 specific type (transfer??), is it?

Reply:
Yep, it's the number of bytes necessary for a trade. And transfers are normally a couple of bytes less so it works well for transfers as well (though if extra addresses are put onchain a transfer can actually make use of almost all 68 bytes, but that should be uncommon).